### DISCUSSION TITLE
Movie Title vs Movie Title imbedded into the mkv file

### DISCUSSION THREAD

**Discussion by Estarna** [Komet/MediaElch#1397] [Ideas]
MKV files have a specific field named "File Title" (picture 1) which seems to be a recognized by many tools, inluding Windows (picture 2).

If MediaElch can read the information from the file, i.e. codec, audio, sub etc,...

Could it read the "File Title" embedded in the video file and show it somewhere like the name field?

Thanks!

-Eric
![1](https://user-images.githubusercontent.com/20680326/150451318-07aead0c-0303-4731-ad77-9c078bdb494f.JPG)
![2](https://user-images.githubusercontent.com/20680326/150451327-2611809e-e982-44b7-a5c6-72c5cd429890.JPG)

**Comment by Estarna** [comment]
First pic is from MKVtoolnix
second is from Windows file explorer

**Comment by bugwelle** [comment]
Hi,

that's an interesting topic. MediaElch does not use the MKV's title, only those for resolution.

It's definitely interesting for movie directories without sub-directories.

Regards,
Andre

**Reply by Estarna** [reply]
Hi again,

It would be an interesting feature, say with an optional enable/disable option.
 
In my case, I usually have specific information there as the version of the film, i.e Extended/Unrated/Cut etc...
This way, if .nfo is altered/lost this information will stay there as it is already written in the mkv file.

If there is no data in the movie "File title" field, MediaElch could go for the regular movie name as it is actually.

Regards,

-Eric
